content/Coronavirus’ Impact on Business/Index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Coronavirus’ Impact on Business - Save Yours!
+date: 2020-09-09T16:00:00.284Z
+description: How to save your business during Coronavirus pandemic
+author: Abeer Albada
+author_company: Zaat.dev
+author_twitter: AbeerAlbada
+author_github: abeeralbada
+image: ./Coronavirus_Impact_on_Business.png
+tags: Business, Coronavirus, Web Development, Software Solutions
+issue_number: 10
+---
```

Comment:
You are missing the `author_title`. Should had point to that in the last in review. 
Sorry for the hassle.